server/svix-server/src/queue/redis.rs

```diff
@@ -462,36 +462,34 @@ impl TaskQueueReceive for RedisQueueConsumer {
 
             // There is no way to make it await a message for unbounded times, so simply block for a short
             // amount of time (to avoid locking) and loop if no messages were retreived
-            let resp = loop {
```

Comment:
Why did the redis code even need to change?

Reply:
With the loop it only broke upon successfully receiving a task. Now it delivers an empty `Vec` after the `BLOCK` period on Redis such that the worker is guaranteed to shutdown eventually.

Reply:
Cool, that makes sense, so just the other comment.